packages/ef-runtime-client/src/ComponentRegistry/ComponentRegistry.ts

```diff
@@ -1,17 +1,22 @@
+export interface IComponentInfo {
```

Comment:
This is conflicting for main because I used the naming EFComponentInfo instead of just ComponentInfo. I did that to differentiate it from say a preact Component.
Also, The believe the js and the css should be optional to give the developers the ability to override only one of them and not both.

Reply:
May be we can discuss prefixing our types with `EF` for example.